src/main/resources/environments/dev.properties

```diff
@@ -1,6 +1,27 @@
 env=dev
-
+# ~~~~~~~~~~~~~~~~~~~~ <<< WebDriver versions >>> ~~~~~~~~~~~~~~~~~~~~~
+chrome.driver.version=101.0.4951.67
+edge.driver.version=""
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# ~~~~~~~~~~~~~~~~~~~~ <<< WebDriver configuration >>> ~
+# possible solutions: chrome, firefox, edge, safari, provided
+# provided type enable custom webdriver configuration
+# webdriver.provided.type should be equal to desired browser name
+webdriver.driver=chrome
+webdriver.provided.type=firefox
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #testRail properties
 testRail.endPoint=https://kovaloleksandra.testrail.io/
 testRail.username=oleksandra_koval@epam.com
-testRail.password=KJv0CdvX0u9arx0jaC2D
\ No newline at end of file
+testRail.password=KJv0CdvX0u9arx0jaC2D
+
+# ~~~~~~~~~~~~~~~~~~~~ <<< Sauce Lab >>> ~~~~~~~~~~~~~~~~~~~~~
+
+SAUCE_USERNAME=oauth-kulyasaleksandra-cb8f8
+SAUCE_ACCESS_KEY=5805de78-2c54-423a-bf5e-bbd008467f6f
+SAUCE_URL=https://oauth-kulyasaleksandra-cb8f8:5805de78-2c54-423a-bf5e-bbd008467f6f@ondemand.eu-central-1.saucelabs.com:443/wd/hub
```

Comment:
partially duplicates with SAUTH_USERNANE and _ACCESS_KEY. Please replace with 'ondemand.eu-central-1.saucelabs.com:443/wd/hub' and build the final Url